packages/testing-toolkit/src/index.ts

```diff
@@ -0,0 +1,244 @@
+import { fireEvent, screen, waitFor } from '@testing-library/react';
+import { userEvent } from '@testing-library/user-event';
+
+export async function inputText(
+  canvasElement: HTMLElement,
+  linkId: string,
+  text: string | boolean | number
+) {
+  const questionElement = await findByLinkId(canvasElement, linkId);
+
+  const input =
+    questionElement?.querySelector('input') ?? questionElement?.querySelector('textarea');
+
+  if (!input) {
+    throw new Error(`Input or textarea was not found inside ${`[data-linkid=${linkId}] block`}`);
+  }
+
+  fireEvent.change(input, { target: { value: text } });
+
+  // Here we await for debounced store update
+  await new Promise((resolve) => setTimeout(resolve, 500));
+}
+export async function checkCheckBox(canvasElement: HTMLElement, linkId: string) {
+  const questionElement = await findByLinkId(canvasElement, linkId);
+  const input =
+    questionElement?.querySelector('input') ?? questionElement?.querySelector('textarea');
+
+  if (!input) {
+    throw new Error(`Input or textarea was not found inside ${`[data-linkid=${linkId}] block`}`);
+  }
+
+  fireEvent.click(input);
+
+  // Here we await for debounced store update
+  await new Promise((resolve) => setTimeout(resolve, 500));
+}
+
+export async function inputFile(
+  canvasElement: HTMLElement,
+  linkId: string,
+  files: File | File[],
+  url: string,
+  filename: string,
+  data_test_url: string,
```

Comment:
why do we need these two attrs? I don't like passing the all the time, they must be hardcoded in this function